src/generator.js

```diff
@@ -2,57 +2,119 @@ import { getEnums, getMetaData, getModels } from "@openactive/data-models";
 import { constants as fsConstants, promises as fs } from "fs";
 import fsExtra from "fs-extra";
 import path from "path";
-import request from "sync-request";
+import request from "then-request";
```

Comment:
Is this something that is used across the files? `npm install`ing doesn't seem to add it as dependency and I can't see it referenced anywhere else - I believe you use `axios` instead?

Reply:
Good spot! I switched out their sync-request with then-request, then switched again to axios. Forgot to remove the import.